### PR TITLE
[Turbo] Modernize code (ppp, types..)

### PR DESCRIPTION
**Pull request by smnandre**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  |no
| Issues        | Fix #...
| License       | MIT

### CODE REVIEW

File: src/Turbo/src/Bridge/Mercure/Broadcaster.php [hunk 1]
```diff
@@ -40,17 +40,15 @@ final class Broadcaster implements BroadcasterInterface
...
     public const TOPIC_PATTERN = 'https://symfony.com/ux-turbo/%s/%s';
 
-    private $name;
-    private $hub;
-
-    /** @var ExpressionLanguage|null */
+    /**
+     * @var ExpressionLanguage|null
+     */
     private $expressionLanguage;
```
Comment (Kocal) [inline src/Turbo/src/Bridge/Mercure/Broadcaster.php]: Can't we type it without PHPDoc? (note the assignment)
```suggestion
    private ExpressionLanguage|null $expressionLanguage = null;
```

File: src/Turbo/src/Broadcaster/TwigBroadcaster.php [hunk 2]
```diff
@@ -21,20 +21,15 @@
...
-    {
-        $this->broadcaster = $broadcaster;
-        $this->twig = $twig;
-        $this->templatePrefixes = $templatePrefixes;
-        $this->idAccessor = $idAccessor ?? new IdAccessor();
+    public function __construct(
+        private BroadcasterInterface $broadcaster,
+        private Environment $twig,
+        private array $templatePrefixes = [],
+        private IdAccessor $idAccessor = new IdAccessor(),
```
Comment (wouterj) [inline src/Turbo/src/Broadcaster/TwigBroadcaster.php]: This is a BC break, the argument is no longer nullable.

File: src/Turbo/src/Bridge/Mercure/Broadcaster.php (hunk 1 above)
Comment (smnandre) [reply src/Turbo/src/Bridge/Mercure/Broadcaster.php]: Well spoted. I set the value in the constructor.

File: src/Turbo/src/Broadcaster/TwigBroadcaster.php (hunk 2 above)
Comment (smnandre) [reply src/Turbo/src/Broadcaster/TwigBroadcaster.php]: Absolutely, thank you. fixed

**Comment by kbond** [comment]
Thank you Simon.

**Pull request merged**